docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,357 @@
+# Getting Started
+This document is intended as a "getting started" guide. The audience are developers looking to modify or *publish* the packages to the Universe. This document is written in tutorial and walk-through format. The goal is to help you "get started". It does not go into great depth on some of the conceptual or inner details. This guide aims at making a user familiar with the concepts of what a Package is and what are the roles of Marathon and Universe in the package life cycle.
+
+
+## Prerequisites
+Before starting this guide, make sure you have met the following conditions.
+
+### Library dependencies
+* [DC/OS CLI](https://dcos.io/docs/latest/cli/install/) installed and configured.
+* [jq](https://stedolan.github.io/jq/download/) is installed in your environment.
+* python3 in your environment.
+* Docker is installed.
+
+### Access requirements
+* Access to a running [DC/OS](https://dcos.io/docs/latest/overview/what-is-dcos/).
+* The Universe Server needs to be built and run in a location accessible by the DC/OS Cluster. There is no other way to test your package otherwise.
+* The Marathon server needs to have access to the Docker Hub registry which has your Universe server. In our guide, we use Docker Hub.
+
+## Required nomenclature
+This guide assumes you are familiar with the basics of the concepts mentioned below. Advanced user can skip this section and jump to [Create a package](#create-a-package). If you are new to DC/OS, we recommend reading this sections along with the external links provided.
+
+
+### What is Universe ?
+The Universe is a DC/OS package repository that contains services like Spark, Cassandra, Jenkins, and many others. It allows users to install these services with a single click from the DC/OS UI or by a simple `dcos package install package_name` command from the DC/OS CLI. Many community members have already submitted their own packages to the Universe, and we encourage anyone interested to get involved with package development! It is a great way of contributing to the DC/OS ecosystem and allows users to easily get started with your favorite package.
```

Comment:
Change the install command to either drop `package_name` (i.e. `dcos package install`), or make it explicit that it's an argument: `dcos package install <package-name>`.